### PR TITLE
Setup prettier

### LINKED ISSUE #34: Implement a Code Formatter
## Is your feature request related to a problem? Please describe.

Right now, we rely on ESLint rules to stick to a code style convention for our development workflow. However, this does not enforce people to adopt those standards bringing noise when doing code review for a PR and introducing different code styles drifting from our standards.

## Describe the solution you'd like

A plausible solution would be to implement a pre-commit hook that would take care of this issue automatically before committing any changes to the repo. This would reduce the amount of noise brought by the different code styles from developers. Thus, making code review easier for everyone

## Describe alternatives you've considered

A good contender for this would be to implement [Prettier](https://prettier.io/) ✨ which would take care of it.

## Additional context

We would need to define a good set of rules to best fit our configuration

### PR DESCRIPTION
**Pull request by manuelurenah**
### Description

* Adds dependencies to run Prettier on staged files
* Adds Husky for pre-commit hooks
* Modifies ESlint rules
* Closes #34 

### How it works

1. Write code without following any styling conventions 👩‍💻 
1. Make a commit ✔️ 
1. Watch the pre-commit hook kick in and Prettier works its magic ✨ 
1. See a pretty styled code base 💅

### CODE REVIEW

File: app/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "trailingComma": "none",
+  "singleQuote": true,
+  "printWidth": 120
```
Comment (james-owen) [inline app/.prettierrc]:
```suggestion
  "printWidth": 100
```
The docs suggest a line length / printWidth of 80, but we could split the difference here and go for 100. https://prettier.io/docs/en/options.html#print-width

File: app/package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "compression": "^1.7.2",
     "connect-redis": "^3.3.3",
     "date-fns": "^1.29.0",
+    "dotenv": "^6.2.0",
```
Comment (PedroAlbR) [inline app/package.json]: We should keep this as a dev dependency

File: app/components/meta-authors/model.js [hunk 3]
```diff
@@ -6,15 +6,16 @@ module.exports.save = (ref, data) => {
   data.authors = data.authors || [];
   // Normalize "authors" value; if saved from a Kiln form, it will be of the form
   // [{text: string}].
-  data.authors = data.authors
-    .map(author => typeof author === 'string' ? author : _get(author, 'text', ''));
+  data.authors = data.authors.map(author =>
```
Comment (james-owen) [inline app/components/meta-authors/model.js]: Could this `data.authors.map` be combined with the below on line 19?

Comment (manuelurenah) [reply app/components/meta-authors/model.js]: I believe they serve different purposes cause one is for the `save` method and the other for the `render` method

**Pull request merged**